index.js

```diff
@@ -810,7 +811,10 @@ module.exports = function (log, indexesPath) {
             })
           )
         } else {
-          const opts = { live: true, gt: seq }
+          const opts =
+            seq === -1
+              ? { live: true, gt: indexes['offset'].seq }
+              : { live: true, gt: seq }
```

Comment:
if `seq === -1`, it means we didn't detect a seq, so instead of doing stream over `seq > -1` (which gives old values, and that might make some sense, except see toPullStream operator, it already runs jitdb.paginate for the old values, and after that, jitdb.live, so we really don't need old values)

Reply:
Ah yes